content/issue-1/weihanglo.md

```diff
@@ -0,0 +1,131 @@
+
+On the other hand, Cargo has many subcommands, with the first-class subcommands being treated differently from the third-party subcommands. The first-class subcommands, such as Cargo build and Cargo tree, can use the internal API freely, but the third-party subcommands cannot. That leads to a situation that first-class subcommands haven't "eaten their own dog food." To address this, we need to find a way to modularize the code and improve it over time, which make the API interface more friendly and extensible.
+
+These are two of the chanllges I am aware of the Cargo team currently facing. It's like a chicken-egg problem. When open-source project grows, contributors may step down or burn out, and new contributors get in. However, the code base becomes increasingly monolithic and difficult for new contributors to understand. To address this, we need to lower the barrier for new contributors to get on board and become reviewers or maintainers. Even if not everyone wants to become a maintainer, they can still follow the trail to learn and bring values. That's the key to succeed for a open source project.
+"""
+```
+
+**Does the Cargo team have any specific plans or goals for the coming year or two? Can you give us a sneak peek into what we can expect from Cargo in the near future?**
+
+```quote
+author = "Weihang Lo"
+content = """
+The Cargo team is trying to write their own roadmap, but it's challenging due to differing opinions and use cases among users. Currently, there's a lack of clear roadmap, but the top priority for the next year is turn the sparse index registry on by default, which will help speed up CI pipelines when it becomes stable in 1.68. However, this requires more communication and collaboration with the infrastructure team and the crates.io team, so personally I don't expect it will become the default right after it hits stable.
+
+As an open-source project, every maintainer in the Cargo team is a volunteer and free to contribute as they wish. We don't set any expectation on a single person. The minimal requirements are attending the meeting regularly and doing FCP reviews.
+
+There are many ongoing projects, such as integration of the new dependency resolver pubgrubs, modularizing Cargo, and collaborate with with private registry providers to figure out the needs of registry autentications. Some other are exploring ideas on nested workspaces or making Cargo release process more smooth. Also some area like parallsing libtest is under consideration. However, `cargo test` depends on litest and may need an edition breaking change for a different mechanism of sharing test environments.
```

Comment:
Is this the pubgrubs? https://github.com/pubgrub-rs/pubgrub

Reply:
Exactly!